backend/server/route/simulation.js

```diff
@@ -4,6 +4,7 @@ const db = require("../db");
 const { auth, headers } = require("../middleware");
 const { httpStatusCode } = require("../constant");
 const { createInvalidResponse } = require("../utils");
+const pages = require("./pages");
 
 router.post(
   "/create",
```

Comment:
This wasn't the URI for this endpoint in the docs. Should be `/`.

Reply:
I haven't checked if this pull-request is following the docs. Most of them must correspond with the docs since I referenced the docs while writing the codes, but I will double-check and fix it.